newspaper/parsers.py

```diff
@@ -247,7 +246,7 @@ def getAttribute(cls, node, attr=None):
         if attr:
             attr = node.attrib.get(attr, None)
         if attr:
-            attr = HTMLParser().unescape(attr)
+            attr = unescape(attr)
```

Comment:
Good change - it seems like HTMLParser().unescape(..) is being deprecated in 3.5+ and we will need to use unescape instead